src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        threshold = (int) (table.length * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resize();
+        }
+        putVal(getHash(key), key, value);
     }
 
     @Override
     public V getValue(K key) {
-        return null;
+        Node<K, V> currentNode = table[getHash(key) % table.length];
+        while (true) {
```

Comment:
```suggestion
        while (currentNode != null) {
```

Reply:
Why is it incorrect? Bohdan said that this is not a mistake(in general). I thought that `while (true)` is easier to understand than complex conditions.

Reply:
because this way your solution is shorter and this condition is not over complicated (only 2 boolean conditions inside)